packages/libremap-agent/Makefile

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright 2013 Patrick Grimm <patrick@lunatiki.de>
+# Copyright 2013 André Gaul <gaul@web-yard.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libremap-agent
+PKG_VERSION:=0.1.9
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
+
+
+PKG_SOURCE_URL:=git://github.com/aparcar/libremap-agent-openwrt.git
```

Comment:
Shouldn't it be HTTPs?

Reply:
no

Reply:
Shouldn't it be https://github.com/libremesh/libremap-agent.git ?

Reply:
In order to be able to build it I moved everything to the top folder https://github.com/aparcar/libremap-agent-openwrt/commit/054133cd07b6ca77bba32537ad2141a193a94111

Reply:
Once this is upstream we could change it back